{"links": {"self": {"href": "data/repositories/levitsky/pyteomics/issues/37/comments/51473784.json"}, "html": {"href": "#!/levitsky/pyteomics/issues/37#comment-51473784"}}, "issue": {"links": {"self": {"href": "data/repositories/levitsky/pyteomics/issues/37.json"}}, "type": "issue", "id": 37, "repository": {"links": {"self": {"href": "data/repositories/levitsky/pyteomics.json"}, "html": {"href": "#!/levitsky/pyteomics"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3f6302de-10da-4d3c-b258-0bb6a2368972}ts=74456"}}, "type": "repository", "name": "pyteomics", "full_name": "levitsky/pyteomics", "uuid": "{3f6302de-10da-4d3c-b258-0bb6a2368972}"}, "title": "Add numpress support in mzML"}, "content": {"raw": "Added support for the layered compression.\nAlso, the `transform` step apparently still needed to be amended for numpress output, because if a numpy array is fed to `np.frombuffer` with a non-matching dtype, the data get corrupted.", "markup": "markdown", "html": "<p>Added support for the layered compression.\nAlso, the <code>transform</code> step apparently still needed to be amended for numpress output, because if a numpy array is fed to <code>np.frombuffer</code> with a non-matching dtype, the data get corrupted.</p>", "type": "rendered"}, "created_on": "2019-04-03T22:59:07.446155+00:00", "user": {"display_name": "Lev Levitsky", "uuid": "{eb44325f-4ee0-4e0b-a27c-f2ea23122a56}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Beb44325f-4ee0-4e0b-a27c-f2ea23122a56%7D"}, "html": {"href": "https://bitbucket.org/%7Beb44325f-4ee0-4e0b-a27c-f2ea23122a56%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/a2593c44c42429c503d2e5e9e307e241d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsLL-6.png"}}, "nickname": "levitsky", "type": "user", "account_id": "557058:986c547b-c50a-40b3-948a-29b4a93b7b30"}, "updated_on": null, "type": "issue_comment", "id": 51473784}